features/orders/view_orders.feature

```diff
@@ -20,8 +20,8 @@ Scenario: view correct order info
   Then I should see the following details for that order:
   | content                           |
   | Purchaser: Tom Foolery            |
-  | Comments: Pickup by: Al Foolery   |
   | Gift? No                          |
   | Order total: $19.00 (Credit card) |
-
+  And I should not see the following details for that order:
```

Comment:
Be careful with negative expectations. If the page shows a random comment or a different comment or even an error message, this step will still pass.  In this case it's less of an issue so you don't need to chagne it, but in general if you are checking for the *absence* of a field, you can also use `expect(page).not_to have_selector(some_css_expression)` or something similar.

Reply:
Got it. I'll change it now.